# Generated by Django 3.0.7 on 2020-07-01 13:41

from django.conf import settings
import django.contrib.postgres.fields.jsonb
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        ('trip_app', '0001_initial'),
    ]

    operations = [
        migrations.AddField(
            model_name='user',
            name='profile_image',
            field=models.FileField(blank=True, null=True, upload_to='profile'),
        ),
        migrations.CreateModel(
            name='UserException',
            fields=[
                ('exception_sqn', models.BigAutoField(primary_key=True, serialize=False)),
                ('user_request', django.contrib.postgres.fields.jsonb.JSONField()),
                ('stack_trace', models.TextField()),
                ('log_datetime', models.DateTimeField(auto_now_add=True, db_column='logDateTime')),
                ('status', models.BooleanField(default=False)),
                ('user', models.ForeignKey(db_column='user_sqn', null=True, on_delete=django.db.models.deletion.PROTECT, to=settings.AUTH_USER_MODEL)),
            ],
        ),
    ]
